### ISSUE TITLE
Integrating bluelets with threaded libraries

### ISSUE THREAD

**Issue by mpenning** [sampsyo/bluelet#7]
I am curious whether bluelets has support or plans to support an interface similar to gevent.threadpool.  Is this possible?

**Comment by sampsyo** [comment]
No, integrating with a native green threads implementation would likely be quite difficult. Maybe Python's standard asyncio has considered that?
